evennia/contrib/game_systems/achievements/README.md

```diff
@@ -0,0 +1,119 @@
+# Achievements
+
+A simple, but reasonably comprehensive, system for tracking achievements. Achievements are defined using ordinary Python dicts, reminiscent of the core prototypes system, and while it's expected you'll use it only on Characters or Accounts, they can be tracked for any typeclassed object.
+
+## Installation
+
+Once you've defined your achievement dicts in a module, assign that module to the `ACHIEVEMENT_CONTRIB_MODULES` setting in your settings.py
+
+Optionally, you can specify what attribute achievement progress is stored in, with the setting `ACHIEVEMENT_CONTRIB_ATTRIBUTE`. By default it's just "achievements".
+
+There is also a command available to let players check their achievements - `from evennia.contrib.game_systems.achievements.achievements import CmdAchieve` and then add `CmdAchieve` to your default Character and/or Account cmdsets.
+
+#### Settings examples
+
+One module:
+
+```python
+# in server/conf/settings.py
+
+ACHIEVEMENT_CONTRIB_MODULES = "world.achievements"
+```
+
+Multiple modules, with a custom-defined attribute:
+
+```python
+# in server/conf/settings.py
+
+ACHIEVEMENT_CONTRIB_MODULES = ["world.crafting.achievements", "world.mobs.achievements"]
+ACHIEVEMENT_CONTRIB_ATTRIBUTE = "achieve_progress"
+```
+
+A custom-defined attribute including category:
+
+```python
+# in server/conf/settings.py
+
+ACHIEVEMENT_CONTRIB_MODULES = "world.achievements"
+ACHIEVEMENT_CONTRIB_ATTRIBUTE = ("achievements", "systems")
+```
+
+## Usage
+
+### Defining your achievements
+
+This achievement system is designed to use ordinary dicts for the achievement data - however, there are certain keys which, if present in the dict, define how the achievement is progressed or completed.
+
+- **name** (str): The searchable name for the achievement. Doesn't need to be unique.
```

Comment:
Is this correct? In the `get_achievement` function, the `key` used to acquire the achievement is the variable name and not this. Since this `name` does _not_ need to be unique, it seems like a poor choice for a 'searchable' key?

Edit: Reading on, I see the `search_achievement` function does search by this field. This probably needs to be explicitly mentioned here, since the variable key is used everywhere else.

Reply:
It's addressed in the search_achievement section. This section is just supposed to define what the fields are, not how to use them; using them is in the usage section.